src/ui/RadarMap.ts

```diff
@@ -155,6 +155,15 @@ class RadarMap extends maplibregl.Map {
     }
     this.on('styledata', onStyleLoad);
   }
+
+  fitToMarkers(fitBoundsOptions: maplibregl.FitBoundsOptions = { padding: 50 }, overrideMarkers?: RadarMarker[]) {
+    const bounds = new maplibregl.LngLatBounds();
+    const markers = overrideMarkers || this._markers;
+    markers.forEach(marker => {
+      bounds.extend(marker.getLngLat());
+    });
+    this.fitBounds(bounds, fitBoundsOptions);
```

Comment:
What does this do if markers is empty?

Reply:
added a no-op if markers is empty. Yeah you can pass in animation options into `fitBoundsOptions` https://docs.mapbox.com/mapbox-gl-js/api/map/#map#fitbounds

Reply:
just do `fitMarkers({animate: false})` to disable animations. it uses `map.flyTo` by default